src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -48,6 +49,9 @@ public abstract class BuildTestBase : IClassFixture<SharedBuildPerTestClassFixtu
 
         public static bool IsUsingWorkloads => s_buildEnv.IsWorkload;
         public static bool IsNotUsingWorkloads => !s_buildEnv.IsWorkload;
+        public static string GetNuGetConfigPathFor(string targetFramework) =>
+            Path.Combine(BuildEnvironment.TestDataPath, "nuget8.config"); // for now - we are still using net7, but with
```

Comment:
I'm probably missing something here. The comment says we are using "net7", but `nuget8.config` contains only dotnet8 feed. And the `targetFramework` is ignored. Can you please explain the idea to me?

Reply:
ah, that comment is stale. We are currently building for net7.0, but the workload will have 8.0 packages, so we need dotnet8 feed. I'll drop the comment in a follow up PR. And `nuget7.config` will also go away once 7.0 is released.